docker-compose.yml

```diff
@@ -9,9 +9,48 @@ services:
       POSTGRES_USER: admin
       POSTGRES_PASSWORD: adminpassword
     volumes:
-      - db-data:/var/lib/postgresql/data
+      - pqdb-data:/var/lib/postgresql/data
```

Comment:
I would not change the volume name to avoid breaking changes for those already using the old volume.